src/alpha-lab/multiple-rim.md

```diff
@@ -4,93 +4,166 @@ permalink: /alpha-lab/multiple-rim/
 
-# Map and visualize gaze onto multiple reference images taken from the same environment
+# Map and visualize gaze on multiple reference images taken from the same environment
 
 <TagLinks />
 
 <div class="iframe-container">
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/7iXIFjdkt-Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/CLNDE0VJVng?si=Hsp2C3JYRCIOtKyY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 <br>
 
-In the [Reference Image Mapper](/enrichments/reference-image-mapper/) guide, we learnt how to properly set up a Reference Image Mapper enrichment, with a <i>single</i> reference image. However, there are some cases in which it would be useful to map gaze onto <i>multiple</i> reference images taken from the same environment - for example, moving in a room while interacting with certain parts of it.
-
 ::: tip
-Before continuing, ensure you are familiar with the [Reference Image Mapper](/enrichments/reference-image-mapper) enrichment. Check out [this explainer video](https://www.youtube.com/watch?v=ygqzQEzUIS4&t=56s) for reference.
+Level-up your Reference Image Mapper workflow to extract insights from participants freely exploring their environment!
 :::
 
-## Reference Images
-
-First, we will take pictures of the areas and/or furniture of the room we are interested in.
-
-| <img src="../media/alpha-lab/cupboard-img.png"/>       | <img src="../media/alpha-lab/desk-img.png"/> |
-| ------------------------------------------------------ | -------------------------------------------- |
-| <img src="../media/alpha-lab/kitchen-imgs.png"/>       | <img src="../media/alpha-lab/tv-img.png">    |
-| <img src="../media/alpha-lab/kitchen+table-img.jpeg"/> |                                              |
-
-## Scanning recordings
-
-In this guide, we want to map gaze onto different parts of a living room, for this reason, we recorded **two** scanning videos. We chose to use more than one scanning recording because the environment is a bit too big to be effectively scanned just by a single one.
+## Exploring gaze patterns in multiple regions of an environment
+Understanding where people focus their gaze while exploring their environment is an area of interest for researchers in 
+diverse fields, ranging from art and architecture to medical training. Recently, powerful scene recognition tools such as 
+the Reference Image Mapper enrichment in Pupil Cloud have made it possible to map gaze in 3D real-world environments and 
+generate heatmap visualizations. These offer an informative overview of visual exploration patterns and also pave the way 
+for further analysis, such as region of interest analysis.
```

Comment:
This paragraph is kind of long. I would suggest that we break it up or reduce it somehow.